src/Inference/Inference.jl

```diff
@@ -123,11 +123,11 @@ testvalue(::Type{T}) where T = zero(T)
 testvalue(v) = testvalue(typeof(v))
 
 function testvalue(::Type{T}) where T<:AbstractArray{E,N} where {E,N}
-   similar(T,fill(0,N)...)
+   similar(T,tfill(0,Val(N))...)
```

Comment:
`Val(N)` is still type-unstable (unless the compiler is able to leverage that N is a compiler constant here).
Use `Val{N}()` instead.

Reply:
It is the same

Reply:
Since https://github.com/JuliaLang/julia/pull/24362 and Julia 1.0, we don't need to be so careful.